explorer/explorerroutes.go

```diff
@@ -319,8 +319,18 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Initially try to get the transaction information from the "lite" mode
-	// data source, which relies on RPCs to dcrd.
+	inout, ok := r.Context().Value(ctxTxInOut).(string)
+	if !ok {
+		inout = ""
+	}
+
+	ioid, ok := r.Context().Value(ctxTxInOutId).(string)
+	var inoutid int64
+	inoutid, err := strconv.ParseInt(ioid, 10, 0)
```

Comment:
Why first declare `var inoutid int64`?  Also, why the if statement, and why the assignment of 0 (the default value already) to `inoutid`?

Reply:
oops, leftover from some type issue I had during development I'm sure.  I'll remove it.

Top of Tx page is now greatly simplified because the default error values are all ok.
```
inout, _ := r.Context().Value(ctxTxInOut).(string)
ioid, _ := r.Context().Value(ctxTxInOutId).(string)
inoutid, _ := strconv.ParseInt(ioid, 10, 0)
```